src/CostFunction.h

```diff
@@ -28,7 +28,7 @@ class CostFunction
 
 protected:
 	int ls_cnt;
-	int64_t mObs;
+	uint64_t mObs;
```

Comment:
Is using `uint64_t` instead of `int64_t` mainly for the 4-panel hurricane case?

Reply:
I think either int64_t or unit63_t works fine for the 4-panel hurricane case.  The change to an unsigned was just the result of taking a closer look at a number of integers throughout the code.  I can change it back if you would prefer.

Reply:
Thanks for your clarification. As long as we have a consistency of using `uint64_t` everywhere else, I am fine with this change. Also shall we consider changing `int` to `uint32_t` in this case?

Reply:
I think eventually we should convert all int definitions to use the C+11 standard definitions.  However, I would suggest that we do this in a separate PR that does not include any complex code changes.  I would consider that a code cleanup PR versus this which is a functional enhance PR.